src/applications/claims-status/sass/claims-status.scss

```diff
@@ -307,6 +307,7 @@ $color-inactive: #9b9b9b;
   font-weight: bold;
   text-decoration: none;
   margin: 0.5em 0;
+  word-break: keep-all;
```

Comment:
This prevents the phone number from wrapping across two lines. This behavior is approved by @emilywaggoner.

Reply:
It looks like we should set up a ticket to better organize the styles, it seems like edu styles shouldn't be affected by claims styles? Also, does this change affect any claims behavior? If so, we should document that as well via screenshot in this PR for approval.

Reply:
Yes, the Sass organization could use a lot of love, based on my brief time with it. That's a dangerous game to play if there aren't really good visual regression tests in place before starting the process. But it's something that I'd like to do. This change will probably affect other phone numbers on the site. But that is a good thing because a phone number probably shouldn't ever wrap to a new line.

Reply:
Thanks @erikphansen could you please add a comment to this file that we should move these to a shared style file, and out of this application specific file?